example/real-world/Render/OptionsForm.purs

```diff
@@ -72,7 +72,7 @@ renderEnabled state =
         [ css "checkbox"
         , HP.type_ InputCheckbox
         , HP.checked $ F.getInput prx.enable state.form
-        , HE.onChange $ HE.input_ $ F.modify prx.enable (not $ F.getInput prx.enable state.form)
+        , HE.onChange $ HE.input_ $ F.modify prx.enable not
```

Comment:
These sorts of "toggle" functions are now possible in Formless @whoadave @crcornwell